package.json

```diff
@@ -84,10 +81,6 @@
     },
     "build": {
       "patterns": [
-        "burf-base",
-        "burf-theme",
-        "burf-customizations",
-        "burf-tools",
```

Comment:
Can you add this back? This is used to have a watch command that publishes changes in those folders by running `yalc:changed` script